bootloader/src/pyi_launch.c

```diff
@@ -358,6 +358,46 @@ pyi_launch_extract_binaries(ARCHIVE_STATUS *archive_status)
     return retcode;
 }
 
+// #if defined(LAUNCH_DEBUG) && defined(WINDOWED)
+static void 
+_handle_system_exit(PyObject *exception, PyObject *value, PyObject *tb) {
+    int exitcode = 0;
+    
+    FATALERROR("fetching exception");
+    PI_PyErr_Fetch(&exception, &value, &tb);
+    FATALERROR("exception fetched");
+    if (!PI_PyErr_GivenExceptionMatches(exception, PI_PyExc_SystemExit)) {
```

Comment:
@danyeaw @bjones1 @htgoebel you're better at C than I am. I've stuck FATALERROR calls everywhere to debug, and the bootloader just crashes on this line.

I'm building with the following command: `pyinstaller test.py --onefile -d bootloader --clean -w`

And test.py is:
```python
raise SystemExit(13)
```